src/backend/gl/src/info.rs

```diff
@@ -237,28 +246,57 @@ pub fn get(gl: &gl::Gl) -> (Info, Capabilities, PrivateCaps) {
         max_texture_size: get_usize(gl, gl::MAX_TEXTURE_SIZE),
         max_patch_size: if tessellation_supported { get_usize(gl, gl::MAX_PATCH_VERTICES) } else {0},
 
-        instance_base_supported:           info.is_version_or_extension_supported(4, 2, "GL_ARB_base_instance"),
-        instance_call_supported:           info.is_version_or_extension_supported(3, 1, "GL_ARB_draw_instanced"),
-        instance_rate_supported:           info.is_version_or_extension_supported(3, 3, "GL_ARB_instanced_arrays"),
-        vertex_base_supported:             info.is_version_or_extension_supported(3, 2, "GL_ARB_draw_elements_base_vertex"),
-        srgb_color_supported:              info.is_version_or_extension_supported(3, 2, "GL_ARB_framebuffer_sRGB"),
-        constant_buffer_supported:         info.is_version_or_extension_supported(3, 1, "GL_ARB_uniform_buffer_object"),
+        instance_base_supported:           info.is_supported(Some((4, 2)),
```

Comment:
I was wondering why the logic there was so complex, I didn't want to cause trouble for someone more versed in the intricacies of the copy buffer extension.
I suppose in the case that there were sufficient difference between the NV and ARB versions it wouldn't be a boolean value.

I don't think that representing GL version number as a floating point is a good idea the [docs](https://www.khronos.org/opengl/wiki/Get_Context_Info#OpenGL_version_number) define it as two integers GL_MAJOR_VERSION and GL_MINOR_VERSION.
I get your point though, I've created a simple macro that arguably makes the code more expressive.